jest.setup.ts

```diff
@@ -0,0 +1,29 @@
+import Color from './src/util/color';
+
+expect.extend({
```

Comment:
Can we do it without this complication? While I get the point behind it, I would like to avoid these special extenstions, or better yet have this as part of the test utils so if we decide to migrate to a different test framework it will be easier.

Reply:
This is the kind of test utils that, in the event of migrating from jest, would still require modification regardless of whether these utils are defined as an extension to `expect` or as separate functions.
The great advantage of defining them as an extension to jest is diff printing when tests are failing.
But it's not my project, and if you don't want custom machers I'll convert it into a separate test utils function (where I'll be forced to use jest specific syntax anyway).

Reply:
Yes, it would be better to have this written in the test utils so that there's no need for the extra configuration and preferably simpler methods.

Reply:
Moved to test utils file.

Reply:
Thanks, see my new comment about suggested location.